cli/src/alluxio.org/cli/processes/job_workers.go

```diff
@@ -0,0 +1,149 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package processes
+
+import (
+	"path"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+)
+
+var JobWorkers = &JobWorkersProcess{
+	BaseProcess: &env.BaseProcess{
+		Name: "job_workers",
+	},
+}
+
+type JobWorkersProcess struct {
+	*env.BaseProcess
+}
+
+func (p *JobWorkersProcess) SetEnvVars(envVar *viper.Viper) {
+	return
+}
+
+func (p *JobWorkersProcess) Base() *env.BaseProcess {
+	return p.BaseProcess
+}
+
+func (p *JobWorkersProcess) StartCmd(cmd *cobra.Command) *cobra.Command {
+	cmd.Use = p.Name
+	return cmd
+}
+
+func (p *JobWorkersProcess) StopCmd(cmd *cobra.Command) *cobra.Command {
+	cmd.Use = p.Name
+	return cmd
+}
+
+func (p *JobWorkersProcess) Start(cmd *env.StartProcessCommand) error {
```

Comment:
the `Start` of the processes also looks the same... maybe we can write a function like:
```golang
func (RemoteProcess) Run(nodes []string, cmdline string)
```
and all the processes can wrap the function to reduce duplication. and the `Start` and the `Stop` can both call the same function.

Reply:
i also notice there is a lot of duplicate code. if we identify the differences between each, then all of them should share the same common code.

besides `all` and `local`, all other commands follow a similar format of
- read the file with list of hostnames
- ssh into each hostname to run a command

we should parameterize the file to read and the command to run into a generalized function that repeats this operation for each of the commands